angular/valueacessor.directive.d.ts

```diff
@@ -0,0 +1,12 @@
+import { TextValueAccessor } from "nativescript-angular/forms/value-accessors/text-value-accessor";
+/**
+ * The accessor for setting a selectedIndex and listening to changes that is used by the
+ * {@link NgModel} directives.
+ *
+ *  ### Example
+ *  ```
+ *  <MaskedTextField [(ngModel)]="model.test">
```

Comment:
Don't think this example should reference MaskedTextField

Reply:
ooops, my bad. I'll fix it tomorrow. I borrowed part of the doc from the maskedtextfield